Web support?

**Feature Request**
<!-- Provide details of what you would like added. -->
Web browsers have a network logger. The only thing that this library should do is "Nothing"!
When i use this library with `react-native-web`,i got this error:
```sh
Failed to compile.
/Users/xxx/xxx/xxx/node_modules/react-native/Libraries/Network/XMLHttpRequest.js
Module not found: Can't resolve './RCTNetworking' in '/Users/xxx/xxx/SellerCenterApp/node_modules/react-native/Libraries/Network'
```
I finally solve this error by **DELETE all references of this library** when i am building for the web.


**Additional context**
<!-- Add any other context or screenshots about the feature request here. -->
On the web platform, this library should do nothing, because the browsers have their logging already.

--- a/src/index.web.tsx
+++ b/src/index.web.tsx
@@ -1,0 +1,16 @@
+import { StartNetworkLoggingOptions } from './types';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const startNetworkLogging = (options?: StartNetworkLoggingOptions) => {
+  console.warn('startNetworkLogging is not implemented on this platform');
+};
+
+export const getRequests = () => [];
+
+export const clearRequests = () => {};
+
+export { getBackHandler } from './backHandler';
+
+export { ThemeName } from './theme';
+
+export default () => null;

